src/pages/Time.js

```diff
@@ -10,19 +10,23 @@ import { getCurrentTimeEntries } from '../selectors/time';
 
 type TimeType = {
   entries: Array<timeType>,
+  settings: {
+    rounding?: string,
+    roudningDown?: string,
+  },
 };
 
-function Time({ entries }: TimeType) {
+function Time({ entries, settings }: TimeType) {
   return (
     <div style={{ paddingLeft: '1%', paddingRight: '1%' }}>
       <DateRange />
-      <ThymeTable entries={entries} />
+      <ThymeTable entries={entries} settings={settings} />
     </div>
   );
 }
 
 function mapStateToProps(state) {
-  return { entries: getCurrentTimeEntries(state) };
+  return { entries: getCurrentTimeEntries(state), settings: state.settings };
```

Comment:
I'd prefer to only pass the rounding settings where needed, not the whole settings object. If the settings object were to grow you'd pass in a lot more than you'd maybe want.